src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/2.sql

```diff
@@ -1542,3 +1873,6 @@ AS
         
     COMMIT TRANSACTION
 GO
+
+COMMIT TRANSACTION
```

Comment:
Just for my understanding, what's the purpose of the nested transactions?

Reply:
So I have wrapped up all the SQL statements in a transaction(given the statements are transactional) for lesser recovery points. So the inner transaction is for DeleteExtendedQueryTag stored proc and while outer transaction is for multiple statements.